js/stores/message_store.js

```diff
@@ -3,30 +3,47 @@ var assign = require('object-assign');
 var AppDispatcher = require('../dispatcher/app_dispatcher');
 var ActionTypes = require('../constants/constants').ActionTypes;
 
-var _messages = [];
+var _successfulMessages = [];
```

Comment:
can we call this `_savedMessages`? just that "successful" doesn't seem relevant for existing messages, only ones you've attempted to save...